Library/Homebrew/requirements/apr_dependency.rb

```diff
@@ -3,12 +3,15 @@
 class AprDependency < Requirement
   fatal true
   default_formula "apr"
+  default_formula "apr-util"
 
+  satisfy { MacOS::CLT.installed? || Formula["apr-util"].installed? }
   satisfy { MacOS::CLT.installed? || Formula["apr"].installed? }
```

Comment:
A requirement can't have multiple default_formula or satisfy calls, the second will just overwrite the first.